### ISSUE TITLE
Uncaught AttributeError in sendchat.py line 30

### ISSUE THREAD

**Issue by martinjo** [paul-gauthier/aider#1635]
Aider version: 0.55.0
Python version: 3.9.14
Platform: macOS-14.5-arm64-arm-64bit
Python implementation: CPython
Virtual environment: No
OS: Darwin 23.5.0 (64bit)
Git version: git version 2.39.5 (Apple Git-154)

An uncaught exception occurred:

```
Traceback (most recent call last):
  File "aider", line 8, in <module>
    sys.exit(main())
  File "main.py", line 694, in main
    io.tool_output()
  File "base_coder.py", line 730, in run
    def get_input(self):
  File "base_coder.py", line 773, in run_one
    return
  File "base_coder.py", line 1183, in send_message
    content += "\n^C KeyboardInterrupt"
  File "base_coder.py", line 1863, in auto_commit
  File "repo.py", line 101, in commit
    else:
  File "repo.py", line 186, in get_commit_message
    continue
  File "sendchat.py", line 38, in wrapper
    retry_exceptions(),
  File "sendchat.py", line 30, in retry_exceptions
    litellm.llms.anthropic.chat.AnthropicError,
AttributeError: module 'litellm.llms.anthropic' has no attribute 'AnthropicError'

```

**Comment by fry69** [comment]
Thank you for filing this issue.

The fix is in the latest version (`0.57.1`), please update aider with
```
aider --upgrade
```

**Comment by paul-gauthier** [comment]
Thanks for trying aider and filing this issue.

This looks like a duplicate of #1278. Please see the comments there for more information, and feel free to continue the discussion within that issue.

I'm going to close this issue for now. But please let me know if you think this is actually a distinct issue and I will reopen this issue.
